app/views/leaderboard.scala.html

```diff
@@ -6,7 +6,13 @@
 @leaderboardStatsThisWeek = @{UserStatTable.getLeaderboardStats(10, "weekly")}
 @leaderboardStatsByOrg = @{UserStatTable.getLeaderboardStats(10, "overall", true)}
 @leaderboardStatsOrg = @{UserStatTable.getLeaderboardStats(10, "overall", false, UserOrgTable.getAllOrgs(user.get.userId).headOption)}
+@import play.api.Play
```

Comment:
Move these imports up with the import at the top of the file

Reply:
Sounds good, done!